sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/executable/SonarTSCoreBundleFactory.java

```diff
@@ -23,14 +23,18 @@
 import org.sonar.api.batch.BatchSide;
 import org.sonar.api.batch.ScannerSide;
 import org.sonar.api.config.Configuration;
+import org.sonarsource.api.sonarlint.SonarLintSide;
+
+import static org.sonarsource.api.sonarlint.SonarLintSide.MULTIPLE_ANALYSES;
 
 @BatchSide
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "BatchSide"; it is deprecated. ([squid:CallToDeprecatedMethod](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=feature%2Fuse_new_anotation&issues=AWEjwKv61DTsLFfQjKla&open=AWEjwKv61DTsLFfQjKla)

[ISSUE_KEY]: AWEjwKv61DTsLFfQjKla

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED